Platform/CoffeelakeBoardPkg/AcpiTables/Dsdt/Platform.asl

```diff
@@ -190,15 +190,6 @@ Field (SPRT, ByteAcc, Lock, Preserve) {
   SSMP, 8
 }
 
-//
-// Operational region for ACPI Control (SMI_EN) access
-//
-OperationRegion (SMIN, SystemIO, (ACPI_BASE_ADDRESS + R_ACPI_IO_SMI_EN), 0x4)
-Field (SMIN, AnyAcc, NoLock, Preserve)
-{
-  SMIE, 32
-}

```

Comment:
You could have this code in platform.asl and remove it in TpmSsdt.asl.

Reply:
done